src/solve.jl

```diff
@@ -1,8 +1,9 @@
 function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algType,
               timeseries_init=uType[], ts_init=tType[], ks_init=[];
               d_discontinuities=tType[], dtmax=tType(7*minimum(prob.lags)), dt=zero(tType),
-              kwargs...) where {uType,tType,isinplace,
-                                algType<:AbstractMethodOfStepsAlgorithm,lType}
+              saveat=tType[], tstops=tType[], save_idxs=nothing,
```

Comment:
As mentioned above, one could of course just filter e.g. components or time steps afterwards. But I thought it might be better to mostly use the already existing methods in OrdinaryDiffEq.jl, instead of implementing these things in a completely different way for DDEs. It's just that these keywords are not usable at the moment, and since I would like to retrieve only specific components at specific time points  I came up with this first idea.